go.mod

```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-10
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.3-0.20220228024614-c785686563e8
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-10...c785686563e8

Reply:
`pending txs` here mean txs that have passed the `antehandler` checks and stay in mempool?

Reply:
yes

Reply:
so it seems both normal cosmos-sdk txs and etherum txs could pass `antehandler`:

https://github.com/tharsis/ethermint/blob/main/app/ante/ante.go#L39-L44

```go
		txWithExtensions, ok := tx.(authante.HasExtensionOptionsTx)
		if ok {
			opts := txWithExtensions.GetExtensionOptions()
			if len(opts) > 0 {
				switch typeURL := opts[0].GetTypeUrl(); typeURL {
				case "/ethermint.evm.v1.ExtensionOptionsEthereumTx":
					// handle as *evmtypes.MsgEthereumTx
					anteHandler = newEthAnteHandler(options)
				case "/ethermint.types.v1.ExtensionOptionsWeb3Tx":
					// handle as normal Cosmos SDK tx, except signature is checked for EIP712 representation
					anteHandler = newCosmosAnteHandlerEip712(options)
				default:
					return ctx, sdkerrors.Wrapf(
						sdkerrors.ErrUnknownExtensionOptions,
						"rejecting tx with unsupported extension option: %s", typeURL,
					)
				}

				return anteHandler(ctx, tx, sim)
			}
		}
```

Reply:
yes, they are all accepted, just go through different ante handler pipelines.